src/OpenTelemetry.Instrumentation.SqlClient/SqlClientInstrumentationOptions.cs

```diff
@@ -154,6 +156,30 @@ internal void AddConnectionLevelDetailsToActivity(string dataSource, Activity sq
             }
         }
 
+        internal void AddDBUserToActivity(string connectionString, Activity sqlActivity)
+        {
+            if (this.EnableConnectionLevelAttributes)
```

Comment:
Since we are altering the definition of `EnableConnectionLevelAttributes`, could you update its comment to reflect this ?

Reply:
Updated